docarray/document/mixins/mesh.py

```diff
@@ -112,3 +112,32 @@ def load_vertices_and_faces_to_point_cloud(self: 'T', samples: int) -> 'T':
             )
 
         return self
+
+    def load_uris_to_rgbd_tensor(self: 'T') -> 'T':
+        """Load RGB image from :attr:`.uri` of :attr:`.chunks[0]` and the depth image from :attr:`.uri` of :attr:`.chunks[1]` and merge them into :attr:`.tensor`.
+
+        :return: itself after processed
+        """
+        from PIL import Image
+
+        if len(self.chunks) != 2:
+            raise ValueError(
+                f'The provided Document does not have 2 chunks but instead {len(self.chunks)}. To load uris to RGBD tensor, the Document needs to have two chunks, with the first one providing the RGB image uri, and the second one providing the depth image uri.'
+            )
+        for chunk in self.chunks:
+            if chunk.uri == '':
+                raise ValueError(
+                    'A chunk of the given Document does not provide an uri.'
+                )
+
+        rgb_img = np.array(Image.open(self.chunks[0].uri).convert('RGB'))
+        depth_img = np.array(Image.open(self.chunks[1].uri))
+
+        if rgb_img.shape[0:2] != depth_img.shape:
+            raise ValueError(
+                f'The provided RGB image and depth image are not of the same shapes: {rgb_img.shape[0:2]} != {depth_img.shape}'
+            )
+
+        self.tensor = np.append(rgb_img, np.expand_dims(depth_img, axis=2), axis=2)
```

Comment:
should not we use np.stack here ?

Reply:
but cannot you use stack after you are do expand_dim ? I am just not sure about using np.append